plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -90,17 +90,17 @@ open class HtmlRenderer(
                     listOf(
                         BasicTabbedContentType.CONSTRUCTOR,
                         BasicTabbedContentType.TYPE,
-                        BasicTabbedContentType.FUNCTION,
-                        BasicTabbedContentType.PROPERTY
+                        BasicTabbedContentType.PROPERTY,
+                        BasicTabbedContentType.FUNCTION
```

Comment:
Changing the order here doesn't actually do anything, so it's just for consistency